.github/workflows/pull-request-check.yml

```
@@ -35,31 +35,45 @@ jobs:
       - name: Set arch environment variable
-        run: |
-          if [[ ${{matrix.runners}} == 'ubuntu-22.04' ]]; then
-            echo arch="amd64" >> $GITHUB_ENV
-          else
-            echo arch="arm64" >> $GITHUB_ENV
-          fi
+        run: echo "arch=${{ matrix.arch.arch }}" >> $GITHUB_ENV
       - name: Checkout che-code source code
         uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

actionlint (1.7.12)

[error] the runner of "docker/setup-qemu-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)